pinecone/sparse/bm25/bm25-quora.ipynb

```diff
@@ -0,0 +1,2451 @@
+{
```

Comment:
<blockquote>" In the following code, we explore the impact of various alpha values using a sample query."</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pinecone-io/examples/pull/138/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>